src/widgets/about-video/ui/about-video.tsx

```diff
@@ -1,30 +1,30 @@
+import classNames from 'classnames/bind';
 import { WidgetTitle } from '@/shared/ui/widget-title';
+import { videoTitleLocalized } from 'data';
 
-import './about-video.scss';
+import styles from './about-video.module.scss';
 
-type AboutVideoProps = { lang?: 'en' | 'ru' };
+const cx = classNames.bind(styles);
 
-const localizedContent = {
-  en: { title: 'RS School video' },
-  ru: { title: 'Видео RS School' },
-};
+type AboutVideoProps = { lang?: 'en' | 'ru' };
 
 export const AboutVideo = ({ lang = 'en' }: AboutVideoProps) => {
   // Needed to prevent flakiness in screenshot tests
   const isRunningInDev = import.meta.env.DEV;
 
   return (
-    <div className="about-video container">
-      <div className="about-video content">
-        <WidgetTitle mods="lines">{localizedContent[lang].title}</WidgetTitle>
-        <div className="video-wrapper">
-          <div className="video-container">
+    <section className={cx('container')} data-testid="about-video">
+      <article className={cx('content')}>
+        <WidgetTitle mods="lines">{videoTitleLocalized[lang].title}</WidgetTitle>
+        <div className={cx('video-wrapper')}>
+          <div className={cx('video-container')}>
             {isRunningInDev
               ? (
-                  <div className="video-placeholder">Video Placeholder</div>
+                  <div className={cx('video-placeholder')}>Video Placeholder</div>
                 )
               : (
                   <iframe
+                    className={cx('video-frame')}
                     loading="lazy"
                     title="RS School Intro"
                     src="https://www.youtube.com/embed/n4unZLVpnaU"
```

Comment:
could we encapsulate the link in `shared` constants?

Reply:
Done